test/integration/http/http_multi_test.go

```diff
@@ -0,0 +1,465 @@
+		}
+
+		peerGroupRunner, _ := fabricnetwork.PeerGroupRunners(network)
+		peerProcesses = ifrit.Invoke(peerGroupRunner)
+		Eventually(peerProcesses.Ready(), network.EventuallyTimeout).Should(BeClosed())
+
+		By("Joining orderers to channels")
+		for _, channel := range channels {
+			fabricnetwork.JoinChannel(network, channel)
+		}
+
+		By("Waiting for followers to see the leader")
+		Eventually(ordererRunners[1].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("Message from 1"))
+		Eventually(ordererRunners[2].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("Message from 1"))
+		Eventually(ordererRunners[3].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("Message from 1"))
+
+		By("Joining peers to channels")
+		for _, channel := range channels {
+			network.JoinChannel(channel, network.Orderers[0], network.PeersWithChannel(channel)...)
+		}
+
+		peer = network.Peer("Org1", "peer0")
+
+		pathToPrivateKeyBackend := network.PeerUserKey(peer, "User1")
+		skiBackend, err = cmn.ReadSKI(pathToPrivateKeyBackend)
+		Expect(err).NotTo(HaveOccurred())
+
+		pathToPrivateKeyRobot := network.PeerUserKey(peer, "User2")
+		skiRobot, err = cmn.ReadSKI(pathToPrivateKeyRobot)
+		Expect(err).NotTo(HaveOccurred())
+
+		admin, err = client.NewUserFoundation(pbfound.KeyType_ed25519)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(admin.PrivateKeyBytes).NotTo(Equal(nil))
+
+		feeSetter, err = client.NewUserFoundation(pbfound.KeyType_ed25519)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(feeSetter.PrivateKeyBytes).NotTo(Equal(nil))
+
+		feeAddressSetter, err = client.NewUserFoundation(pbfound.KeyType_ed25519)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(feeAddressSetter.PrivateKeyBytes).NotTo(Equal(nil))
+
+		cmn.DeployACL(network, components, peer, testDir, skiBackend, admin.PublicKeyBase58, admin.KeyType)
+		cmn.DeployCC(network, components, peer, testDir, skiRobot, admin.AddressBase58Check)
+		cmn.DeployIndustrial(network, components, peer, testDir, skiRobot,
+			admin.AddressBase58Check, feeSetter.AddressBase58Check, feeAddressSetter.AddressBase58Check)
+	})
+	BeforeEach(func() {
+		By("start robot")
+		robotRunner := networkFound.RobotRunner()
+		robotProc = ifrit.Invoke(robotRunner)
+		Eventually(robotProc.Ready(), network.EventuallyTimeout).Should(BeClosed())
+
+		By("start channel transfer")
+		channelTransferRunner := networkFound.ChannelTransferRunner()
+		channelTransferProc = ifrit.Invoke(channelTransferRunner)
+		Eventually(channelTransferProc.Ready(), network.EventuallyTimeout).Should(BeClosed())
+	})
+	AfterEach(func() {
+		By("stop robot")
+		if robotProc != nil {
+			robotProc.Signal(syscall.SIGTERM)
+			Eventually(robotProc.Wait(), network.EventuallyTimeout).Should(Receive())
+		}
+
+		By("stop channel transfer")
+		if channelTransferProc != nil {
+			channelTransferProc.Signal(syscall.SIGTERM)
+			Eventually(channelTransferProc.Wait(), network.EventuallyTimeout).Should(Receive())
+		}
+	})
```

Comment:
where stop redis

Reply:
fixed. added stopping redis stage to http tests